components/server/ee/src/container-module.ts

```diff
@@ -64,6 +62,8 @@ import { BitbucketServerApp } from "./prebuilds/bitbucket-server-app";
 import { EntitlementService } from "../../src/billing/entitlement-service";
 import { EntitlementServiceChargebee } from "./billing/entitlement-service-chargebee";
 import { BillingModes, BillingModesImpl } from "./billing/billing-mode";
+import { EntitlementServiceLicense } from "./billing/entitlement-service-license";
+import { EntitlementServiceImpl } from "./billing/entitlement-service";
```

Comment:
now re-reading this, it seems odd that ES is in "billing", especially that EntitlementServiceImpl is delegating to different strategies.

Reply:
Jep. Thought about making it `/entitlement/` but could stomach that quickly, so left it here, for now. :shrug: